packages/admin-ui/src/custom-collapsibe/custom-collapsible.stories.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react'
+import type { Meta } from '@storybook/react'
+
+import {
+  CardTitle,
+  CardImage,
+  Card,
+  CardContent,
+  CardHeader,
+  CardInfo,
+} from '../card'
+import { Tag } from '../tag'
+import {
+  IconImageSquare,
+  IconCaretDown,
+  IconCaretRight,
+} from '@vtex/phosphor-icons'
+import { useCollapsible } from './use-collapsible'
+import { Box } from '../box'
+import { Divider } from '../components/Divider'
+
+export default {
+  title: 'admin-ui/CollapsibleCard',
+  component: Card,
+} as Meta
+
+export const SingleCollapsible = () => {
+  const {
+    getToggleProps,
+    getCollapseProps,
+    visible: isExpanded,
```

Comment:
I noticed that within `useCollapsible` we're converting a variable named `isExpanded` to be returned as `visible`. But here we convert from `visible` to `isExpanded`. I feel like it'd be best to choose just one of these two as the default: either keep the original `isExpanded` name or let this one be just `visible`, as it arrives.

Reply:
I would go with `visible` since this syntax is already used in other components (example: Modal and Menu, probably other pop-up components)